elasticdl/python/worker/worker.py

```diff
@@ -262,51 +262,33 @@ def _process_minibatch(
         self._timing.end_record_time("batch_process")
         return min_model_version
 
-    def _process_eval_task(self, task):
+    def _process_evaluation_dataset(self, dataset):
```

Comment:
To better describe this function:
`_process_evaluation_dataset` -> `_process_evaluation_if_exist`.

`evaluation_executed` -> `evaluation_exist`.